src/components/admin/Billing/index.tsx

```diff
@@ -38,32 +36,32 @@ const routeTitle = authenticatedRoutes.admin.billing.title;
 const invoiceCardHeight = TOTAL_CARD_HEIGHT + 5;
 
 function AdminBilling({ showAddPayment }: AdminBillingProps) {
+    usePageTitle({
+        header: routeTitle,
+        headerLink: 'https://www.estuary.dev/pricing/',
+    });
+
     const selectedTenant = useTenantStore((state) => state.selectedTenant);
 
     // Billing Store
-    const hydrated = useBillingStore((state) => state.hydrated);
-    const setHydrated = useBillingStore((state) => state.setHydrated);
+    const [active, setActive] = useBillingStore((state) => [
+        state.active,
+        state.setActive,
+    ]);
```

Comment:
Added in `active` to know when things are loading vs just knowing if we have data to show.